modules/RTC/ScreenObtainer.js

```diff
@@ -254,6 +254,25 @@ const ScreenObtainer = {
         getDisplayMedia(constraints)
             .then(stream => {
                 this.setContentHint(stream);
+
+                // Apply min fps constraints to the track so that 0Hz mode doesn't kick in.
+                // https://bugs.chromium.org/p/webrtc/issues/detail?id=15539
+                if (browser.isChromiumBased()) {
+                    const track = stream.getVideoTracks()[0];
+                    const min = desktopSharingFrameRate.min && desktopSharingFrameRate.min > 0
+                        ? desktopSharingFrameRate.min : SS_DEFAULT_FRAME_RATE;
+                    const contraints = {
```

Comment:
Can you update it so if desktopSharingFrameRate is undefined it will not fail. 
https://github.com/jitsi/lib-jitsi-meet/pull/2383/files#diff-ce2d682b5181dd9a7081e3a971c936cf99e8c6d24644c7dafcd0ee10e85d30f5R230

Reply:
I wonder whether that is the case for failing tests.

Reply:
Hum... 🤷 
```
2023-10-27T18:32:42.135Z [modules/RTC/ScreenObtainer.js] <Object.obtainScreenFromGetDisplayMedia>:  Using getDisplayMedia for screen sharing [object Object]
2023-10-27T18:32:42.137Z [modules/RTC/ScreenObtainer.js] getDisplayMedia error [object Object] [object Object]
2023-10-27T18:32:42.138Z [features/base/tracks] Failed to create local tracks desktop gum.screensharing_user_canceled: User canceled screen sharing prompt: Error
    at new Gt (https://oracle-prtorture.jitsi.net/libs/lib-jitsi-meet.min.js?v=139:2:580699)
    at https://oracle-prtorture.jitsi.net/libs/lib-jitsi-meet.min.js?v=139:2:742285

```

Reply:
You are too fast, I was looking at the test failure. It should be fixed now.